data/campaigns/Eastern_Invasion/scenarios/07a_The_Crossing.cfg

```diff
@@ -368,7 +373,8 @@
                     [then]
                         [message]
                             speaker=Grug
-                            message= _ "Grug say join you will he."
+                            #po: "Grug say join you will he" means "I will join you" in a weird speech pattern. "Gorg and Drog too" means that two other ogres will be joining Gweddry.
```

Comment:
There's a `Gorg` in here which should be `Gork`. Only noticed it because I was thinking "that sounds too close to Grug".

Reply:
Good catch!